code/go/0chain.net/smartcontract/dbs/event/process.go

```diff
@@ -155,6 +156,13 @@ func (edb *EventDb) addStat(event Event) error {
 			return err
 		}
 		return edb.addOrOverwriteValidator(vn)
+	case TagAddMint:
```

Comment:
TagAddMint and TagAddBlock could come in either order, but they will both be populating the same row.

Reply:
naming it update confuses with the capability to change other properties... are we going to allow that?